service-front/module/Application/config/module.config.php

```diff
@@ -550,33 +570,734 @@
             'xid' => 'National identity card',
         ],
         'acceptable_nations_for_id_documents' => [
-            'AT' => 'Austria',
-            'BD' => 'Belgium',
-            'BG' => 'Bulgaria',
-            'HR' => 'Croatia',
-            'CW' => 'Cyprus',
-            'CZ' => 'Czechia',
-            'DK' => 'Denmark',
-            'EE' => 'Estonia',
-            'FI' => 'Finland',
-            'FR' => 'France',
-            'DE' => 'Germany',
-            'GR' => 'Greece',
-            'HU' => 'Hungary',
-            'IE' => 'Ireland',
-            'IT' => 'Italy',
-            'LV' => 'Latvia',
-            'LT' => 'Lithuania',
-            'LU' => 'Luxembourg',
-            'MT' => 'Malta',
-            'NL' => 'Netherlands',
-            'PL' => 'Poland',
-            'PT' => 'Portugal',
-            'RO' => 'Romania',
-            'SK' => 'Slovakia',
-            'SI' => 'Slovenia',
-            'ES' => 'Spain',
-            'SE' => 'Sweden',
+            'AUT' => 'Austria',
+            'BEL' => 'Belgium',
+            'BGR' => 'Bulgaria',
+            'HRV' => 'Croatia',
+            'CYP' => 'Cyprus',
+            'CZE' => 'Czechia',
+            'DNK' => 'Denmark',
+            'EST' => 'Estonia',
+            'FIN' => 'Finland',
+            'FRA' => 'France',
+            'DEU' => 'Germany',
+            'GRC' => 'Greece',
+            'HUN' => 'Hungary',
+            'IRL' => 'Ireland',
+            'ITA' => 'Italy',
+            'LVA' => 'Latvia',
+            'LTU' => 'Lithuania',
+            'LUX' => 'Luxembourg',
+            'MLT' => 'Malta',
+            'NLD' => 'Netherlands',
+            'POL' => 'Poland',
+            'PRT' => 'Portugal',
+            'ROU' => 'Romania',
+            'SKV' => 'Slovakia',
+            'SVN' => 'Slovenia',
+            'ESP' => 'Spain',
+            'SWE' => 'Sweden',
+        ],
+        "supported_countries_documents" => [
```

Comment:
Potentially work for a next ticket: but I think we should be fetching this from the Yoti API rather than hard-coding it, in case they change what documents they accept.

Reply:
It's complicated because OPG is putting additional rules on the Yoti document list according to yesterday's planning. This static array will need to be updated for that anyway.

Reply:
In that case I believe we need to:
- Include passports from non-EU countries in this
- Remove "TRAVEL_DOCUMENT" since that's not on our list of accepted ID

It's probably also worth removing `is_strictly_latin` and `date_from` since we're not using them.

We should also ask Yoti how we can be notified when their supported documents change, to avoid sending people to the Post Office with something that won't be supported.